Item.cs

```diff
@@ -26,25 +25,25 @@ public enum TypeItem
 
         public int _id = 0;
         private string _name;
-        public Shelf? _shelfOfItem { get; set; }
+        //public Shelf? _shelfOfItem { get; set; }
+        public int _idShelf { get; set; }
         public Kashrot _kashrot { get; set; }
         public TypeItem _type { get; set; }
         public DateTime _expiryDate { get; set; }
         public int _size;
 
 
-        public Item(string name, Kashrot kashrot, TypeItem typeFood, DateTime expiryDate, int size, Shelf shelfItem = null)
+        public Item(string name, Kashrot kashrot, TypeItem typeFood, DateTime expiryDate, int size, int idShelf = 0)
         {
             try
             {
                 _id = _nextID++;
```

Comment:
fix this according to guid change. 
the purpose is not managing int id.
instead you can handle GUID a unique id and just assign it.